src/components/EnterpriseContractView/useEnterpriseContractResultFromLogs.tsx

```diff
@@ -15,39 +18,81 @@ export const useEnterpriseContractResultFromLogs = (
 ): [ComponentEnterpriseContractResult[], boolean] => {
   const { namespace, workspace } = useWorkspaceInfo();
   const [taskRun, loaded, error] = useTaskRuns(namespace, pipelineRunName, 'verify');
+  const [fetchTknLogs, setFetchTknLogs] = React.useState<boolean>(false);
   const [ecJson, setEcJson] = React.useState<EnterpriseContractResult>();
   const [ecLoaded, setEcLoaded] = React.useState<boolean>(false);
+  const ecResultOpts = React.useMemo(() => {
+    const podName = loaded && !error ? taskRun[0].status.podName : null;
+    return podName
+      ? {
+          ns: namespace,
+          name: podName,
+          path: 'log',
+          queryParams: {
+            container: 'step-report',
```

Comment:
given the point of #971 was to stop using `report` step, this should probably pull logs from `report-json` step for future use

Reply:
Yes, I will make that change